corehq/apps/app_manager/models.py

```diff
@@ -5262,6 +5265,12 @@ def create_jadjar_from_build_files(self, save=False):
 
                 return jadjar.jad, jadjar.jar
 
+    _timing_context = None
+    def time_build(self):
+        self._timing_context = TimingContext(self.name)
+        return self._timing_context
```

Comment:
hmm, actually I may be able to do this without `self._timing_context`, let me try

Reply:
updated and rebased, since I don't think anyone had looked yet